### PR TITLE
REF: remove always-UTC arg from tz_convert, tz_convert_single

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/_libs/tslibs/tzconversion.pyx [hunk 1]
```diff
@@ -366,93 +366,69 @@ cdef int64_t tz_convert_utc_to_tzlocal(int64_t utc_val, tzinfo tz, bint* fold=NU
...
-        return _tz_convert_tzlocal_utc(val, tz, to_utc=to_utc)
+        return _tz_convert_tzlocal_utc(val, tz, to_utc=False)
     else:
         arr[0] = val
-        return _tz_convert_dst(arr, tz, to_utc=to_utc)[0]
+        return _tz_convert_dst(arr, tz)[0]
 
 
-def tz_convert(int64_t[:] vals, tzinfo tz1, tzinfo tz2):
+def tz_convert(int64_t[:] vals, tzinfo tz):
```
Comment (mroeschke) [inline pandas/_libs/tslibs/tzconversion.pyx:399]: Could we rename this `tz_convert_from_utc` to make the intention clear?

File: pandas/_libs/tslibs/tzconversion.pyx [hunk 2]
```diff
@@ -366,93 +366,69 @@ cdef int64_t tz_convert_utc_to_tzlocal(int64_t utc_val, tzinfo tz, bint* fold=NU
     return _tz_convert_tzlocal_utc(utc_val, tz, to_utc=False, fold=fold)
 
 
-cpdef int64_t tz_convert_single(int64_t val, tzinfo tz1, tzinfo tz2):
+cpdef int64_t tz_convert_single(int64_t val, tzinfo tz):
     """
-    Convert the val (in i8) from timezone1 to timezone2
+    Convert the val (in i8) from UTC to tz
 
     This is a single timezone version of tz_convert
```
Comment (simonjayhawkins) [inline pandas/_libs/tslibs/tzconversion.pyx:373]: is this still required?

File: pandas/_libs/tslibs/tzconversion.pyx (hunk 1 above)
Comment (jbrockmendel) [reply pandas/_libs/tslibs/tzconversion.pyx:399]: sure i guess

File: pandas/_libs/tslibs/tzconversion.pyx (hunk 2 above)
Comment (jbrockmendel) [reply pandas/_libs/tslibs/tzconversion.pyx]: updated

File: pandas/_libs/tslibs/tzconversion.pyx [hunk 3]
```diff
@@ -366,93 +366,69 @@ cdef int64_t tz_convert_utc_to_tzlocal(int64_t utc_val, tzinfo tz, bint* fold=NU
...
-
-    converted = _tz_convert_one_way(vals, tz, to_utc=to_utc)
+    converted = _tz_convert_one_way(vals, tz)
     return np.array(converted, dtype=np.int64)
 
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-cdef int64_t[:] _tz_convert_one_way(int64_t[:] vals, tzinfo tz, bint to_utc):
+cdef int64_t[:] _tz_convert_one_way(int64_t[:] vals, tzinfo tz):
```
Comment (jreback) [inline pandas/_libs/tslibs/tzconversion.pyx:424]: let's rename this too

Comment (jbrockmendel) [reply pandas/_libs/tslibs/tzconversion.pyx]: updated+green

**Comment by jreback** [comment]
thanks @jbrockmendel

**Pull request merged**